tests/album-update.test.js

```diff
@@ -0,0 +1,51 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const db = require("../src/db");
+const app = require("../src/app");
+
+describe("Update Album", () => {
+  let album;
+  let artist;
+  beforeEach(async () => {
+    const artistTable = await db.query("INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *", [
+      "MF DOOM",
+      "rap"
+    ]);
+
+    artist = artistTable.rows[0];
+
+    const albumTable = await db.query("INSERT INTO Albums (name, year, artistId) VALUES ($1, $2, $3) RETURNING *", [
+      "MM..FOOD",
+      2004,
+      artist.id
+    ]);
+
+    album = albumTable.rows[0];
+  });
+
+  describe("PATCH /albums/{id}", () => {
+    it("updates the album and returns the updated record", async () => {
+      const { status, body } = await request(app)
+        .patch(`/albums/${album.id}`)
+        .send({ name: "Updated Album", year: 2022, artistid: artist.id });
+      expect(status).to.equal(200);
```

Comment:
Theres should always be a clear line before you assertions start so easy to see where they begin:
```
        .send({ name: "Updated Album", year: 2022, artistid: artist.id });

      expect(status).to.equal(200);
```